.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -28,22 +29,19 @@ body:
     validations:
       required: true
   - type: textarea
-    id: what-bug
+    id: what-problem
     attributes:
       label: What is the problem you see?
       description: What do you consider to be incorrect and why is is incorrect?
       value: |
-        **Is your feature request related to a problem? Please describe.**
-        A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+        **Please describe the problem you see**
+        A clear and concise description of what the problem is. Ex. I think we need to change [...]
 
         **Describe the solution you'd like**
-        A clear and concise description of what you want to happen.
+        A clear and concise description of what change you propose.
```

Comment:
Add in here ` Consider how the change proposed further connects the specification to the [SDG Digital Investment Framework](https://dial.global/research/sdg-digital-investment-framework/)
`

Reply:
added